plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -347,53 +280,60 @@ func (a *traceflowPlugin) traceflowHandler(request service.Request) (component.C
 	card.AddAction(genGraph)
 
 	graphCard := component.NewCard(component.TitleFromString("Antrea Traceflow Graph"))
-	if a.lastTfName != "" {
+	if lastTf.Name != "" {
 		// Invoke GenGraph to show
 		log.Printf("Generating content from CRD...")
 		ctx := context.Background()
-		tf, err := a.client.OpsV1alpha1().Traceflows().Get(ctx, a.lastTfName, metav1.GetOptions{})
+		tf, err := client.OpsV1alpha1().Traceflows().Get(ctx, lastTf.Name, v1.GetOptions{})
 		if err != nil {
-			log.Printf("Failed to generate content from CRD, lastTfName %v, err: %s", a.lastTfName, err)
-			return component.ContentResponse{}, nil
+			log.Printf("Failed to get latest CRD, last traceflow name: %s, err: %s", lastTf.Name, err)
+			log.Printf("Using traceflow results cache, traceflow name: %s", lastTf.Name)
```

Comment:
I believe we can merge these two logs into one.
@tnqn  We change the logic here to handle the case that Traceflow CRD is cleaned up. But I am not sure what we want to show in this case, a blank graph or a previous traceflow graph. Could you also share your opinion here?

Reply:
I don't have preference on this, both work for me.

Reply:
OK, then we can keep the logic here for now to show the previous trace graph if we can not successfully get the CRD.